zproject/backends.py

```diff
@@ -1696,7 +1696,7 @@ def auth_complete(self, *args: Any, **kwargs: Any) -> Optional[HttpResponse]:
             # though AuthFailed would have been more correct.
             #
             # We have an open PR to python-social-auth to clean this up.
-            logging.info("/complete/apple/: %s", str(e))
+            self.logger.info("/complete/apple/: %s", str(e))
```

Comment:
Also should we change the logger for warning logs too ? @chdinesh1089

Reply:
I think yes if it's an auth backend because with  #15183 we wanted all auth backends to have a different logger so that i would become obvious which backend generated the log (it's discussed [here](https://chat.zulip.org/#narrow/stream/3-backend/topic/auth.20backend.20loggers/near/882092) in czo)
But where are they? I think I've replaced them all and I probably missed this as I was working on apple auth PR and #15183 at the same time)